archeryutils/classifications/agb_indoor_classifications.py

```diff
@@ -180,10 +176,11 @@ def calculate_agb_indoor_classification(
     """
     # Check score is valid
     if score < 0 or score > ALL_INDOOR_ROUNDS[roundname].max_score():
-        raise ValueError(
+        msg = (
             f"Invalid score of {score} for a {roundname}. "
-            f"Should be in range 0-{ALL_INDOOR_ROUNDS[roundname].max_score()}."
+            f"Should be in range 0-{ALL_INDOOR_ROUNDS[roundname].max_score()}.",
```

Comment:
This happens quite a few other times, I didn't catch them all explicitly with comments

Reply:
Yeah, they're causing the tests to fail I think, so I need to sort them.
I have a feeling it is one of the linting "fixes" rather than the formatter, so if I remove whatever causes it we should be OK.

Reply:
Turns out it didn't cause tests to fail, but should now be fixed anyway (see https://github.com/jatkinson1000/archeryutils/pull/81#issuecomment-1979518909)